nvme.c

```diff
@@ -3498,6 +3508,24 @@ static int format(int argc, char **argv, struct command *cmd, struct plugin *plu
 		OPT_END()
 	};
 
+	err = argconfig_parse(argc, argv, desc, opts);
+	if (err)
+		goto ret;
+
+	err = fd = open_exclusive(argc, argv, cfg.force);
```

Comment:
I wonder if we should restrict the exclusive open to 'format'; other functions like 'sanitize' or 'write' will suffer from the same problem.
Care to add the '--force' option to 'sanitize', 'write', 'write-zeroes', and 'write-uncor'?

Reply:
First i wanted to fix issue #1095 only. Once this is accepted my goal was to search for all other functions to add the --force option for them too.